app/models/social-link.js

```diff
@@ -12,22 +12,22 @@ export default ModelBase.extend({
 
   event: belongsTo('event'),
 
-  normalizedName: computed('name', function() {
+  capitalizeName: computed('name', function() {
     // Even though name is required for social links and is non-nullable
     // and non-null name is being sent from API, for some reason, for certain events,
     // this throws an error, so we check first if name exists
     // https://github.com/fossasia/open-event-frontend/issues/4777
 
-    const normalizedName = this.name?.trim().toLowerCase();
-    if (!socialPlatforms.includes(normalizedName)) {
-      return 'globe';
+    const capitalizeName = (s) => {
```

Comment:
Codacy found an issue: ['capitalizeName' is assigned a value but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6232875)